pkg/network/server.go

```diff
@@ -671,7 +671,7 @@ func (s *Server) requestBlocks(p Peer) error {
 func (s *Server) handleMessage(peer Peer, msg *Message) error {
 	s.log.Debug("got msg",
 		zap.Stringer("addr", peer.RemoteAddr()),
-		zap.String("type", string(msg.CommandType())))
+		zap.String("type", string(msg.Command)))
```

Comment:
This is wrong way to do this, it converts `byte` to a byte in `utf-8` string.
Maybe implement `String()` method? (can be generated via `stringer`).